### PR TITLE
Add changes for round trend and some other things

### PR DESCRIPTION
**Pull request by illourr**
Made a variety of changes for fun. If you don't agree with or don't like some of them I'm fine with taking them out!

**Changes:**
- Subtext for `X rounds since last month` on dashboard is based on real data now. It is setup to compare the current month with the previous month, like February vs January. 
  - This made me wonder if the entire dashboard page should be a monthly view by default so all metrics are based on the current month. Could take this a step further and allow seeing data by specific amounts of time (pick a month, last 6 months, custom time window, etc)
- Made `<DropdownMenuPrimitive.Item>` use `cursor-pointer` instead of `cursor-default`. Felt weird for the Dropdown items to not have a `pointer` since they are clickable.
- Prefilled round score data with the par for each hole to make it easier to create new rounds, this may be more of a quality of life change for testing but could also be a good starting point when adding new rounds.
- Swapped the order of the Light mode and Dark mode text. Clicking Light mode now changes theme to Light mode, and clicking Dark mode changes theme to Dark mode instead of it showing the active mode. 

Tested=true

### CODE REVIEW

**Comment by ehermanson** [comment]
> This made me wonder if the entire dashboard page should be a monthly view by default so all metrics are based on the current month. Could take this a step further and allow seeing data by specific amounts of time (pick a month, last 6 months, custom time window, etc)

Yeah totally - I've been going back and forth between defaulting to a month by month view or a rolling 30 day view. Just thinking that the first week or maybe two of a given month might not be very useful. OR instead of defaulting to a timeframe, default to a dashboard w/ the last X rounds (10?). 

But yeah either way, definitely agree and like the thought of making the dashboard configurable by timeframe etc! Kinda like how financial apps often do it.

File: app/root.tsx [hunk 1]
```diff
@@ -341,7 +341,7 @@ const Mode = () => {
 			<div className="flex items-center gap-2 relative">
 				<Sun className="h-[1.2rem] w-[1.2rem] rotate-0 scale-100 transition-all dark:-rotate-90 dark:scale-0" />
 				<Moon className="absolute h-[1.2rem] w-[1.2rem] rotate-90 scale-0 transition-all dark:rotate-0 dark:scale-100" />
-				{theme === Theme.DARK ? 'Dark' : 'Light'} Mode
+				{theme === Theme.LIGHT ? 'Dark' : 'Light'} Mode
```
Comment (ehermanson) [inline app/root.tsx:344]: i never know if these types of UI should show the current state or the "desired" state 😄  - both are confusing to me

File: app/routes/_index/dashboard.tsx [hunk 2]
```diff
@@ -102,21 +107,25 @@ export function Dashboard() {
...
 }
 
-const QuickStat = ({
-	title,
-	stat,
-	trend,
-	icon,
-	trendDirection,
-}: QuickStatProps) => {
+const QuickStat = ({ title, stat, trend, trendDirection }: QuickStatProps) => {
```
Comment (ehermanson) [inline app/routes/_index/dashboard.tsx:123]: i think TS should be complaining about removing the `icon` prop and still using it above

File: app/routes/_index/dashboard.tsx [hunk 3]
```diff
@@ -49,9 +54,9 @@ export function Dashboard() {
 				/>
 				<QuickStat
 					title="Rounds Played"
-					stat="6"
-					trend="2 fewer than this time last month"
-					trendDirection="negative"
+					stat={`${rounds.length}`}
```
Comment (ehermanson) [inline app/routes/_index/dashboard.tsx:55]: we can make the `stat` prop `string | number` - number seems more appropriate anyways 😅

**Review by ehermanson — APPROVED** [review]
awesome, thanks for the updates! couple minor things but then we can 🚀

File: app/root.tsx (hunk 1 above)
Comment (illourr) [reply app/root.tsx:344]: It makes sense to me that, you see the current theme is light or dark (the page is majority that color), and what you click is what you are choosing to swap to. 🤷‍♂️

File: app/routes/_index/dashboard.tsx (hunk 2 above)
Comment (illourr) [reply app/routes/_index/dashboard.tsx:123]: Went ahead and deleted the prop from the interface and from every usage of `<QuickStat />`

File: app/routes/_index/dashboard.tsx (hunk 3 above)
Comment (illourr) [reply app/routes/_index/dashboard.tsx:55]: Updated to be `string | number`

**Comment by illourr** [comment]
> > This made me wonder if the entire dashboard page should be a monthly view by default so all metrics are based on the current month. Could take this a step further and allow seeing data by specific amounts of time (pick a month, last 6 months, custom time window, etc)
> 
> Yeah totally - I've been going back and forth between defaulting to a month by month view or a rolling 30 day view. Just thinking that the first week or maybe two of a given month might not be very useful. OR instead of defaulting to a timeframe, default to a dashboard w/ the last X rounds (10?).
> 
> But yeah either way, definitely agree and like the thought of making the dashboard configurable by timeframe etc! Kinda like how financial apps often do it.

Yeah I'd say pick a default view like last 30 days. 👍  Created an issue to create a top level filter -> #2

**Pull request merged**